src/client/Microsoft.Identity.Client/Internal/Constants.cs

```diff
@@ -41,6 +41,7 @@ internal static class Constants
 
         public const string ManagedIdentityClientId = "client_id";
         public const string ManagedIdentityResourceId = "mi_res_id";
+        public const string ManagedIdentityDefaultClientId = "ManagedIdentityClient";
```

Comment:
```suggestion
        public const string ManagedIdentityDefaultClientId = "ManagedIdentityDefaultClientId";
```

Reply:
please track this comment to add what azure sdk has today.

Reply:
Updated it